src/ResourceNavigationCard.php

```diff
@@ -11,7 +11,7 @@ class ResourceNavigationCard extends Card
      *
      * @var string
      */
-    public $width = '1/2';
+    public $width = 'full';
```

Comment:
Changing this to full causes an issue when you have multiple cards on the resource... nova sort the cards placing the smallest at the top...  and here I intentionally set it to the smallest width possible so this card would always be placed on top of everything

Reply:
Well, I only set it to full because it had conflicts with https://github.com/optimistdigital/nova-tailwind when the width is 1/2. It actually takes half the width then.

Reply:
There is a hack here: https://github.com/dcasia/resource-navigation-tab/blob/master/resources/js/components/ResourceNavigationCard.vue#L69 it is initially set to the smallest size possible.. so nova puts it at the top.. and as soon as the page loads it is changed via js back to full

Reply:
In my current build with my fix loaded from my fork, the tabs still stay on top, btw.

<img width="1054" alt="Screenshot 2022-02-24 at 07 55 13" src="https://user-images.githubusercontent.com/1039100/155460552-47e2ef01-c911-437b-9a93-8327312f064e.png">

Reply:
because it is full width.. try to use some metrics card that is 1/3 they will go above the tabs